backend/postgres/health_postgres.go

```diff
@@ -0,0 +1,64 @@
+// Copyright © 2019 by PACE Telematics GmbH. All rights reserved.
+// Created at 2019/10/18 by Charlotte Pröller
+
+package postgres
+
+import (
+	"time"
+
+	"github.com/go-pg/pg"
+	"github.com/pace/bricks/maintenance/health/servicehealthcheck"
+)
+
+// HealthCheck checks the state of a postgres connection. It must not be changed
+// after it was registered as a health check.
+type HealthCheck struct {
+	state      servicehealthcheck.ConnectionState
+	Pool       *pg.DB
+	CheckWrite bool
```

Comment:
That's why the documentation of this type states that you must not change the health check after it is registered. Even if someone would do it, it would be a false negative, which I think is ok for a configuration issue.

Reply:
Ok, I'm fine with that but still would try to avoid things like this in the future.